project/Build.scala

```diff
@@ -507,4 +512,16 @@ object Build extends sbt.Build {
         )
       )
   )
+
+  lazy val pipeline = Project(
+    id = "gearpump-experiments-pipeline",
+    base = file("experiments/pipeline"),
+    settings = commonSettings ++ myAssemblySettings ++
+      Seq(
+        libraryDependencies ++= Seq(
+          "org.apache.hbase" % "hbase-client" % clouderaHBaseVersion,
+          "org.apache.hbase" % "hbase-common" % clouderaHBaseVersion
```

Comment:
Ok.

So, let's keep pipeline and hbase under experiment for now. We can move them to related folder when both get stable.